docs/en/overview.md

```diff
@@ -1 +1,76 @@
 # Overview
+
+This chapter introduces you to the framework of MMSegmentation, the basic conception of semantic segmentation, and provides links to detailed tutorials about MMSegmentation.
+
+## What is semantic segmentation?
+
+Semantic segmentation is the task of clustering parts of an image together that belong to the same object class.
```

Comment:
```suggestion
Semantic segmentation is an approach detecting, for every pixel, belonging class of the object. For example, when all people in a figure are segmented as one object and background as one object. [Ref](https://en.wikipedia.org/wiki/Image_segmentation)
```
Ref: https://en.wikipedia.org/wiki/Image_segmentation.

Reply:
it might not suit to **semantic** segmentation